### PR TITLE
An example in the taskrun doc is not runnable

### LINKED ISSUE #1855: The provided example for `Task` and `TaskRun` doesn't work
# Expected Behavior
Example will work
# Actual Behavior
Example doesn't work
# Steps to Reproduce the Problem

`kubectl apply -f <on-the-file-below>.yaml`

```yaml
apiVersion: tekton.dev/v1alpha1
kind: Task
metadata:
  name: read-task
spec:
  inputs:
    resources:
      - name: workspace
        type: git
  steps:
    - name: readme
      image: ubuntu
      command:
        - /bin/bash
      args:
        - "cat README.md"
---
apiVersion: tekton.dev/v1alpha1
kind: PipelineResource
metadata:
  name: go-example-git
spec:
  type: git
  params:
    - name: url
      value: https://github.com/pivotal-nader-ziada/gohelloworld
---
apiVersion: tekton.dev/v1alpha1
kind: TaskRun
metadata:
  name: read-repo-run
spec:
  taskRef:
    name: read-task
  inputs:
    resources:
      - name: workspace
        resourceRef:
          name: go-example-git
```
Getting 
```
/bin/bash: cat README.md: No such file or directory
```
# Additional Info

### PR DESCRIPTION
**Pull request by sbwsg**
This commit fixes #1855 - an example in the taskrun doc is not runnable.

Two changes were required to make this example in docs/taskruns.md
runnable:

1. "cat README.md" needed to be updated to "cat workspace/README.md"
because the git resource is checked out to /workspace/workspace

2. The TaskRun needed to be moved from above the resource and task
declarations to be the last resource in the document.

As an added bonus I've updated the example to use the script field
instead of `bash -c $(args)`.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] ~~Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)~~
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.47.0 // indirect
+	cloud.google.com/go/storage v1.0.0
```
Comment (afrittoli) [inline go.mod]: This change seems to be unrelated

File: go.mod [hunk 2]
```diff
@@ -51,7 +52,7 @@ require (
 	golang.org/x/sys v0.0.0-20191119060738-e882bf8e40c2 // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
 	golang.org/x/tools v0.0.0-20191118222007-07fc4c7f2b98 // indirect
-	google.golang.org/api v0.10.0 // indirect
+	google.golang.org/api v0.10.0
```
Comment (afrittoli) [inline go.mod]: Ditto

File: docs/taskruns.md [hunk 3]
```diff
@@ -406,10 +393,20 @@ spec:
   steps:
     - name: readme
       image: ubuntu
-      command:
-        - /bin/bash
-      args:
-        - "cat README.md"
+      script: cat workspace/README.md
```
Comment (afrittoli) [inline docs/taskruns.md]: Nice :)

**Comment by sbwsg** [comment]
> I would prefer to keep unrelated changes into a different PR.

No problem, I've removed the changes to go.mod
